### DISCUSSION TITLE
Size - A combo of w- and h-

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#2084] [Ideas]
I've been in cases where I have like a square image and need to adjust `w-16` to something else. Then I also need to change `h-16` to the new value.

```html
<img src="" class="w-16 h-16">
```

...will become...

```html
<img src="" class="w-24 h-24">
```

Instead, in this case it would be nice to have a `size` class.

```html
<img src="" class="size-16">
```

I think I've seen this approach to a draft in native CSS as well.

**Comment by adamwathan** [comment]
Chatted about this a while back here:

https://github.com/tailwindlabs/tailwindcss/pull/1288

**Reply by jenstornell** [reply]
Alright, I did not see that one.

It's not the number of letters in the class name that makes this feature valueble. It's the fact that when prototyping in the browser, it's a nag to change two values instead of one. It takes the double amount of time to get it right.

I respect that you want to wait until there is a native CSS implementation first.
